### PR TITLE
Added a property and new resources

### PR DESCRIPTION
**Pull request by JohnSharpe**
Modules are now properties in AA databases
latest_backup and latest_import retrieval behaviour has been included

### CODE REVIEW

File: internal/service.go [hunk 1]
```diff
@@ -72,39 +72,83 @@ func (a *api) WaitForResource(ctx context.Context, id string, resource interface
 		return err
 	}
 
-	err = json.Unmarshal(*task.Response.Resource, resource)
+	return json.Unmarshal(*task.Response.Resource, resource)
```
Comment (greg-oc) [inline internal/service.go:75]: does this always return a response of type `error`? My Go is a little rusty so it's probably fine

File: client.go [hunk 2]
```diff
@@ -65,6 +70,9 @@ func NewClient(configs ...Option) (*Client, error) {
 		Database:     databases.NewAPI(client, t, config.logger),
 		Subscription: subscriptions.NewAPI(client, t, config.logger),
 		Regions:      regions.NewAPI(client, t, config.logger),
+		LatestBackup: latest_backups.NewAPI(client, t, config.logger),
+		LatestImport: latest_imports.NewAPI(client, t, config.logger),
+		//Pricing:      pricing.NewAPI(client),
```
Comment (greg-oc) [inline client.go:75]: Should "Pricing" be commented out? (also on line 35)

**Review by greg-oc — COMMENTED** [review]
LGTM, just have a couple of questions and then I'll approve

File: client.go (hunk 2 above)
Comment (JohnSharpe) [reply client.go:75]: Ooh it's going to be part of an upcoming commit, although I might release this project before then so I'll remove, good catch!

File: internal/service.go (hunk 1 above)
Comment (JohnSharpe) [reply internal/service.go:75]: Yeah this is go's approach to exception handling - return nil if everything went well. Almost all non-trivial method signatures have an error return-type somewhere.
This line probably used to be something like this:
```
err = json.Unmarshal(*task.Response.Resource, resource)
if err != nil {
  return err
}
return nil
```

Comment (greg-oc) [reply internal/service.go:75]: Cheers, it's coming back to me, looks good in that case

**Pull request merged**